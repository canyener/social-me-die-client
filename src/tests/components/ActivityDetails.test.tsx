// import React from 'react'
// import { shallow } from 'enzyme'

// import ActivityDetails from '../../features/activities/details/ActivityDetails'
// import { fakeActivities } from '../fixtures/seed'

// test('Should render ActivityDetails correctly', () => {
//   const wrapper = shallow(
//     <ActivityDetails
//       setEditMode={() => { }}
//       setSelectedActivity={() => { }}
//     />
//   )

//   expect(wrapper).toMatchSnapshot()
// })

// test('Should call setEditMode with true when Edit button is clicked', () => {

//   const setEditModeSpy = jest.fn()

//   const wrapper = shallow(
//     <ActivityDetails
//       setEditMode={setEditModeSpy}
//       setSelectedActivity={() => { }}
//     />
//   )
//   wrapper.find('Button').first().simulate('click')
//   expect(setEditModeSpy).toHaveBeenCalledWith(true)
// })

// test('Should call setSelectedActivity with null when Cancel button is clicked', () => {
//   const setSelectedActivitySpy = jest.fn()

//   const wrapper = shallow(
//     <ActivityDetails
//       setEditMode={() => { }}
//       setSelectedActivity={setSelectedActivitySpy}
//     />
//   )

//   wrapper.find('Button').at(1).simulate('click')

//   expect(setSelectedActivitySpy).toHaveBeenCalledWith(null)
// })

test('Will be refactored', () => {
  const message = 'Will be refactored after mobx store implementation'
  expect('Will be refactored after mobx store implementation').toBe(message)
})

export {}