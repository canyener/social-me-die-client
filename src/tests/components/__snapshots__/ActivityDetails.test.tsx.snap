// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ActivityDetails correctly 1`] = `
<Card
  fluid={true}
>
  <Image
    as="img"
    src="/assets/categoryImages/Fake Category 1.jpg"
    ui={false}
    wrapped={true}
  />
  <CardContent>
    <CardHeader>
      Fake Activity 1
    </CardHeader>
    <CardMeta>
      <span>
        12/31/1969, 7:00:01 PM
      </span>
    </CardMeta>
    <CardDescription>
      Fake description
    </CardDescription>
  </CardContent>
  <CardContent
    extra={true}
  >
    <ButtonGroup
      widths={2}
    >
      <Button
        as="button"
        basic={true}
        color="blue"
        onClick={[Function]}
      >
        Edit
      </Button>
      <Button
        as="button"
        basic={true}
        color="grey"
      >
        Cancel
      </Button>
    </ButtonGroup>
  </CardContent>
</Card>
`;
